jsonnet/kube-prometheus/components/kube-state-metrics.libsonnet

```diff
@@ -12,6 +12,12 @@ local defaults = {
     limits: { cpu: '100m', memory: '250Mi' },
   },
 
+  kubeRbacProxyMain: {
+    resources+: {
+      limits+: { cpu: '40m' },
+      requests+: { cpu: '20m' },
+    },
```

Comment:
L101 to L104 should be removed as they are overriding what is set here.

Reply:
sorry forget to remove L101 to L104 :grimacing: